config/config.go

```diff
@@ -154,11 +154,11 @@ type StorageOptionsConf struct {
 	// Maximum TLS version that is supported.
 	// Options: ["1.0", "1.1", "1.2", "1.3"].
 	// Defaults to "1.3".
-	MaxVersion string `json:"max_version"`
+	MaxVersion string `json:"tls_max_version"`
```

Comment:
You'll have to change the field too, it ends up in a config `env` var, TYK_STORAGE_TLSMAXVERSION...